platform/innovium/sonic-platform-modules-cameo/esc601-32q/modules/mcp3425_smbus.c

```diff
@@ -0,0 +1,432 @@
+/*
+ * mcp3422.c - driver for the Microchip mcp3421/2/3/4/5/6/7/8 chip family
```

Comment:
Why not use the mcp3422.c which is part of the Linux kernel?

Reply:
Because of I2C bus controller is only SMbus(I801) mode on our board. In mcp3422 linux native driver will call "i2c_check_functionality" and return "EOPNOTSUPP", so we do a work around in it.

Reply:
The mcp3425_smbus.c is based on Linux mcp3422.c to fit our dedicated SMbus requirement. Any further comment on this issue?

Reply:
@lguohan to comment.